### PR TITLE
Libraries update for Stable Diffusion (#335)

### PR DESCRIPTION
**Pull request by pgmpablo157321**
* compliance checker update for StableDiffusion

* RCP checker update for StableDiffusion

* renamed the stable diffusion rules files to match the benchmark name

* added "stable_diffusion" to relevant configs, renamed rcp file

### CODE REVIEW

**Comment by pgmpablo157321** [comment]
@nv-rborkar @ahmadki Please review. The previous PR was not pointing to the maste branch

File: mlperf_logging/compliance_checker/README.md [hunk 1]
```diff
@@ -87,6 +89,7 @@ The following fields are optional:
 - `REQ` - specifies the requirement regarding occurrence. Possible values :
     - `EXACTLY_ONE` - current key has to appear exactly once
     - `AT_LEAST_ONE` - current key has to appear at least once
+    - `AT_LEAST_N_TIMES(n)` - current key has to appear at least n times
```
Comment (xyhuang) [inline mlperf_logging/compliance_checker/README.md:92]: can we name this AT_LEAST(n)

File: mlperf_logging/compliance_checker/mlp_compliance.py [hunk 2]
```diff
@@ -147,6 +147,11 @@ def parse_alternatives(self, string):
         alternatives = in_pharentises.split(',')
         return [s.strip() for s in alternatives]
 
+    def parse_at_least_n_times(self, string):
```
Comment (xyhuang) [inline mlperf_logging/compliance_checker/mlp_compliance.py:150]: ditto.

File: mlperf_logging/compliance_checker/training_3.1.0/closed_stable_diffusion.yaml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    CHECK: " v['value'] >= 0.0 "
+
+- KEY:
+    NAME:  opt_learning_rate_warmup_steps
+    REQ:   EXACTLY_ONE
+    CHECK: " v['value'] >= 0 "
+
+- KEY:
+    NAME: aggregated_eval_accuracy
+    REQ: AT_LEAST_N_TIMES(2)
```
Comment (xyhuang) [inline mlperf_logging/compliance_checker/training_3.1.0/closed_stable_diffusion.yaml:69]: ditto.

**Review by xyhuang — APPROVED** [review]

**Pull request merged**